### DISCUSSION TITLE
Out of sync audio/frozen images on Ubuntu

### DISCUSSION THREAD

**Discussion by rileymd88** [GerryDazoo/Slinger#337] [General]
Hi there,

Is anyone else experiencing issues like this? Sometimes the stream will start fine but eventually it will always have these issues. Here is some more info:
I have setup the `/etc/sysctl.conf` file correctly
I have a consistent upload speed (10-15mb/s)
Running on an old laptop. While loading a stream the CPU usage rarely goes over 10% and it has 8GB of RAM which is also barely being used
I tried turning down the resolution but I still had the same issues
In the logs I do not see any kind of glaring error or warning:
```
Slinginfo  "350/500/M1/M2" 10 30 192.168.0.4 5201 3072 10
8080 Starting Stream for  MY_IP
Connecting... ('192.168.0.4', 5201) Control
8080 Selecting Video Source 1
Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 22, 23, 24, 25, 26, 27, 28, 29, 31, 32, 33, 34, 35, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 53, 54, 55, 56, 57, 58, 59, 60]
VideoParameters: Resolution= 10 FrameRate= 30 VideoBandwidth= 1500 VideoSmoothness= 55 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.0.4', 5201) Stream
8080 Stream started at 11/20/2022, 21:39:48.740  741 0
11/20/2022, 21:39:56.272  8080 Stream Terminated for  MY_IP [Errno 32] Broken pipe
8080 Shutting down connections
8080 Logging Out
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
11/20/2022, 21:43:04.934   Streaming request from (MY_IP)
```

Any other settings/configs I should be looking at?

Thanks

**Comment by rileymd88** [comment]
Moving to discussions!

**Comment by GerryDazoo** [comment]
We really need all the server output to help diagnose. Try a different media player than VLC.

**Reply by rileymd88** [reply]
Hi @GerryDazoo 

Which server output specifically would help? Do you mean the log file found here: `/tmp/sling.log`
